### PR TITLE
udapp fixes

### PR DESCRIPTION
**Pull request by LianaHus**
- fixing badges for fork alert
- fixing paddings in function buttons
- btns are now disabled if input is required but not filled
- some small refactoring and fixes in some themes
- CSS style classes -> bootstrap
- coding style fixes

### CODE REVIEW

File: libs/remix-ui/run-tab/src/lib/components/contractDropdownUI.tsx [hunk 1]
```diff
@@ -201,12 +202,19 @@ export function ContractDropdownUI (props: ContractDropdownProps) {
...
-      address = ethJSUtil.toChecksumAddress(address)
+    if (address == '') return
+    try {
+      if (!ethJSUtil.isValidChecksumAddress(address)) {
+        props.tooltip(checkSumWarning())
+        address = ethJSUtil.toChecksumAddress(address)
+      }
+      props.loadAddress(loadedContractData, address)
+    } catch(e) {
+      console.log("Invalid Address input: ", e)
```
Comment (Aniket-Engg) [inline libs/remix-ui/run-tab/src/lib/components/contractDropdownUI.tsx:213]: remove log

**Review by Aniket-Engg — COMMENTED** [review]
1. Not sure if `No compiled contracts` can be more visible. Also in tooltip text, `AtAddress'` can be replace with `At Address`

Screenshot 2023-07-06 at 2 24 43 PM

**Comment by LianaHus** [comment]
> 1. Not sure if `No compiled contracts` can be more visible. Also in tooltip text, `AtAddress'` can be replace with `At Address`
done
> Screenshot 2023-07-06 at 2 24 43 PM
> 
> 2. `At address` button looks enabled even if no address is provided
done
> 3. About the badges, I think we decided to remove `compiled by Remix` for non-remixd connection
done
> 4. Disabling the method name on no input parameters can be confusing for some users, can we have a tooltip when method button is disabled
we had. I'll check what's broken there

File: apps/remix-ide/src/app/tabs/locales/en/udapp.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   "udapp.contractOptionsTitle1": "Please compile *.sol file to deploy or access a contract",
   "udapp.contractOptionsTitle2": "Select a compiled contract to deploy or to use with At Address.",
   "udapp.contractOptionsTitle3": "Select and compile *.sol file to deploy or access a contract.",
-  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with AtAddress.'",
+  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with At Address.'",
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/locales/en/udapp.json:29]: I think `'` from the end can still be removed

File: apps/remix-ide/src/app/tabs/locales/es/udapp.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
   "udapp.contractOptionsTitle1": "Please compile *.sol file to deploy or access a contract",
   "udapp.contractOptionsTitle2": "Select a compiled contract to deploy or to use with At Address.",
   "udapp.contractOptionsTitle3": "Select and compile *.sol file to deploy or access a contract.",
-  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with AtAddress.'",
+  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with At Address.'",
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/locales/es/udapp.json:28]: same here

File: apps/remix-ide/src/app/tabs/locales/fr/udapp.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
   "udapp.contractOptionsTitle1": "Please compile *.sol file to deploy or access a contract",
   "udapp.contractOptionsTitle2": "Select a compiled contract to deploy or to use with At Address.",
   "udapp.contractOptionsTitle3": "Select and compile *.sol file to deploy or access a contract.",
-  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with AtAddress.'",
+  "udapp.contractOptionsTitle4": "When there is a compiled .sol file, choose the  contract to deploy or to use with At Address.'",
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/locales/fr/udapp.json:28]: and here

**Review by Aniket-Engg — COMMENTED** [review]
If I compile a contract and select another in editor, it shows contracts dropdown empty:

Screenshot 2023-07-10 at 3 36 41 PM

Not sure if it is intentional

**Comment by Aniket-Engg** [comment]
If I provide an invalid type value for constructor params, deploy button is enabled. When I click on `Deploy` button, nothing happens. User has no clue what is wrong

**Comment by LianaHus** [comment]
> to use with AtAddress.'

same on live. will fix it in second pr

**Comment by Aniket-Engg** [comment]
> If I provide an invalid type value for constructor params, deploy button is enabled. When I click on `Deploy` button, nothing happens. User has no clue what is wrong

fixed by https://github.com/ethereum/remix-project/pull/3912

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**